src/server/worldserver/worldserver.conf.dist

```diff
@@ -3241,6 +3241,15 @@ ShowKickInWorld = 0
 
 ShowMuteInWorld = 0
 
+#
+#     Mute.AddAfterLogin.Enable
+#        Description: Add mute time after login player.
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+#
+
+Mute.AddAfterLogin.Enable = 1
```

Comment:
I think we can just name it Mute.AddAfterLogin?

Reply:
Hm, maybe

Reply:
only reasoning I have is as the 2 options are Enable and Disable

Reply:
Okay, i replaced